notifications.bs

```diff
@@ -227,16 +241,22 @@ section of HTML. [[!HTML]]
 
 <p>User agents are expected to honor the Unicode semantics of the text of a
 <a lt="concept notification">notification</a>'s
-<a lt="concept title">title</a> and <a>body</a>. Each is expected to be
-treated as an independent set of one or more bidirectional algorithm paragraphs
-when displayed, as defined by the bidirectional algorithm's rules P1, P2, and
-P3, including, for instance, supporting the paragraph-breaking behavior of
-U+000A LINE FEED (LF) characters. For each paragraph of the
-<a lt="concept title">title</a> and <a>body</a>, the
-<a lt="concept notification">notification</a>'s
+<a lt="concept title">title</a>, <a>body</a> and each of the <a>actions</a>.
+Each is expected to be treated as an independent set of one or more
+bidirectional algorithm paragraphs when displayed, as defined by the
+bidirectional algorithm's rules P1, P2, and P3, including, for instance,
+supporting the paragraph-breaking behavior of U+000A LINE FEED (LF) characters.
+For each paragraph of the <a lt="concept title">title</a>, <a>body</a> and each
+of the <a>actions</a>, the <a lt="concept notification">notification</a>'s
```

Comment:
Just the action's "title" attribute, BiDi doesn't have to care about the "action" value.